### PR TITLE
Rebranding tests folder

### PR DESCRIPTION
**Pull request by ArrowsFlying**
Things left to look into that I did not touch:

tests/core/io/test_marshalls.h:149 - Godot name in number of characters
tests/core/io/test_packet_peer.h - file does not exist
tests/core/io/test_pck_packer.h:114 - length
tests/core/string/test_string.h
- :303 titleized change?
- :295 & :308 keep Godoh ?
tests/core/string/test_string.h:1783 - TEST_CASE("[String] SHA1/SHA256/MD5") line count different, no checks on urls
image files to change:
- tests/data/images/icon.bmp
- tests/data/images/icon.jpg
- tests/data/images/icon.png
- tests/data/images/icon.webp

### CODE REVIEW

File: tests/core/io/test_marshalls.h [hunk 1]
```diff
@@ -141,7 +141,7 @@ TEST_CASE("[Marshalls] Floating point double precision decoding") {
 }
 
 TEST_CASE("[Marshalls] C string encoding") {
-	char cstring[] = "Godot"; // 5 characters
+	char cstring[] = "Blazium"; // 5 characters
```
Comment (jss2a98aj) [inline tests/core/io/test_marshalls.h:144]: "Blazium" is a little bit longer than 5 characters.

**Review by jss2a98aj — COMMENTED** [review]
Do we actually need to rebrand tests? None of the changes here are going to be seen by end users.

**Comment by ArrowsFlying** [comment]
> Do we actually need to rebrand tests? None of the changes here are going to be seen by end users.

Only did as instructed :D

File: tests/core/io/test_marshalls.h (hunk 1 above)
Comment (ArrowsFlying) [reply tests/core/io/test_marshalls.h:144]: On line 149 there's ca character size check. Checks every character. Didn't touch

**Comment by Bioblaze** [comment]
We're closing this, if you would like to contact us on Discord we can pick it back up.
